classrooms/models.py

```
@@ -1,3 +1,82 @@
+from django.utils.text import slugify
+from core.models import BaseModel
+from accounts.models import User
 
-# Create your models here.
+
+class Classroom(BaseModel):
+    title = models.CharField(max_length=255)
+    short_introduction = models.CharField(max_length=500)
+    course_description = models.TextField()
+    course_image = models.ImageField(upload_to='course_images/', default='course_images/default.jpg', null=True, blank=True)
+    preview_video_link = models.URLField(max_length=200, null=True, blank=True)
```

Ruff (0.8.2)

Avoid using `null=True` on string-based fields such as `URLField`

(DJ001)